### ISSUE TITLE
A11y_Azure Tools Storage Explorer_FileShares_Filesharetable_ScreenReader: NVDA is not announcing the correct table data while navigating to each cell of the table. 

### ISSUE THREAD

**Issue by v-inanda** [microsoft/AzureStorageExplorer#4985]
### GitHubTags:
#A11yTCS;#A11ySev2;#A11yWCAG;#DesktopApp;#Azure Tools Storage Explorer;#BM-AzureToolsStorageExplorer-Win32-Sep2021;#ScreenReader;#NVDA;#WCAG1.3.1;#Benchmark;#Win32

### Environment Details:​
Application Name: Azure Tools Storage Explorer
Windows Version: Windows 10

### Note:
No issue in Narrator Screen Reader scan mode

### Repro Steps:​
1.Open NVDA
2.Open Azure Tools Storage Explorer
3.Tab till "File Shares" of the selected subscription and expand it.
4.Click on any existing File share.
5.Navigate to Filesharetable under the tool bar.
6.Verify whether NVDA is announcing the correct table data while navigating to each cell of the table or not.

### Actual Result:
NVDA is not announcing the correct table data while navigating to each cell of the table.

### Expected Result: ​
NVDA should announce the correct table data while navigating to each cell of the table.

### User Impact:
Screen reader user will not get correct  information about the data in the table as it is not announced by screen reader.

### Attachments:
![A11y_Azure Tools Storage Explorer_FileShares_Filesharetable_ScreenReader NVDA is not announcing the correct table data while navigating to each cell of the table  ](https://user-images.githubusercontent.com/91525824/135086072-c28cb9bf-5bc1-4065-b355-eb58dca3e288.gif)
![No issue in Narrator screenreader scan mode](https://user-images.githubusercontent.com/91525824/135268130-ab2d5e49-5e9d-4213-81a0-f22ddbd14116.gif)

**Comment by v-inanda** [comment]
Regressed the bug in the Version :1.22.0-dev
AzCopy version: 10.12.2
The issue is fixed so closing the bug. Please find the attachment for reference.
![4985_NVDA_Fix](https://user-images.githubusercontent.com/91525824/138219200-b3bcb741-352f-4a84-85a8-fa94ca81e235.gif)

**Comment by Sobi261** [comment]
#Closed;
